### PR TITLE
Add parameter EnableRbcAuthorization for 'New-AzKeyVault' and 'Update-AzKeyVault'

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/KeyVault/KeyVault/ChangeLog.md [hunk 1]
```diff
@@ -18,7 +18,10 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+* Added support to enable RBAC authorization [#10557]
+    - `New-AzKeyVault` supports generating a key with RBAC authorization
+    - `Update-AzKeyVault` supports enabling/disabling a key to authorize data actions by RBAC
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/ChangeLog.md:23]:
```suggestion
* Added support for RBAC authorization [#10557]
```
We should be concise when writing change logs

File: src/KeyVault/KeyVault/KeyVault.format.ps1xml [hunk 2]
```diff
@@ -365,6 +365,10 @@
               <ListItem>
                 <Label>Enabled For Disk Encryption?</Label>
                 <PropertyName>EnabledForDiskEncryption</PropertyName>
+              </ListItem>
+			  <ListItem>
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/KeyVault.format.ps1xml:369]:
```suggestion
              <ListItem>
```
Wrong indentation. Please always use spaces.

File: src/KeyVault/KeyVault/Models/VaultManagementClient.cs [hunk 3]
```diff
@@ -193,6 +195,10 @@ public PSKeyVault GetVault(string vaultName, string resourceGroupName, ActiveDir
                 && updatedPurgeProtectionSwitch.Value)
                 properties.EnablePurgeProtection = updatedPurgeProtectionSwitch;
 
+            // Update EnableRbacAuthorization when specified, otherwise stay current value
```
Comment (isra-fel) [inline src/KeyVault/KeyVault/Models/VaultManagementClient.cs:198]: nice comment

**Review by isra-fel — DISMISSED** [review]
LGTM

**Comment by isra-fel** [comment]
Key vault team said this feature is still in preview.
So, do not merge.

**Review by isra-fel — APPROVED** [review]

**Comment by isra-fel** [comment]
Feature is ready, so we can merge this PR

**Pull request merged**